### ISSUE TITLE
recursive clone does not work

### ISSUE THREAD

**Issue by Nicholaswogan** [AER-RC/MT_CKD#3]
Recursive clone fails for some permissions reason.

```
(base) nicholas@Nicholass-Air Downloads % git clone --recursive https://github.com/AER-RC/MT_CKD.git
Cloning into 'MT_CKD'...
remote: Enumerating objects: 330, done.
remote: Counting objects: 100% (21/21), done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 330 (delta 19), reused 19 (delta 19), pack-reused 309
Receiving objects: 100% (330/330), 1.03 MiB | 4.45 MiB/s, done.
Resolving deltas: 100% (111/111), done.
Submodule 'LBLRTM' (https://github.com/AER-RC/LBLRTM.git) registered for path 'LBLRTM'
Submodule 'aer_rt_utils' (https://github.com/AER-RC/aer_rt_utils.git) registered for path 'aer_rt_utils'
Cloning into '/Users/nicholas/Downloads/MT_CKD/LBLRTM'...
remote: Enumerating objects: 4376, done.        
remote: Counting objects: 100% (128/128), done.        
remote: Compressing objects: 100% (94/94), done.        
remote: Total 4376 (delta 76), reused 73 (delta 33), pack-reused 4248        
Receiving objects: 100% (4376/4376), 6.50 MiB | 5.50 MiB/s, done.
Resolving deltas: 100% (3125/3125), done.
Cloning into '/Users/nicholas/Downloads/MT_CKD/aer_rt_utils'...
remote: Enumerating objects: 15, done.        
remote: Counting objects: 100% (15/15), done.        
remote: Compressing objects: 100% (3/3), done.        
remote: Total 15 (delta 12), reused 15 (delta 12), pack-reused 0        
Receiving objects: 100% (15/15), 5.82 KiB | 5.82 MiB/s, done.
Resolving deltas: 100% (12/12), done.
Submodule path 'LBLRTM': checked out '78ab6e374bec049591a0edbd9a4d1bd65f6d7616'
Submodule 'aer_rt_utils' (git@github.com:AER-RC/aer_rt_utils.git) registered for path 'LBLRTM/aer_rt_utils'
Submodule 'cross-sections' (git@github.com:AER-RC/cross-sections.git) registered for path 'LBLRTM/cross-sections'
Cloning into '/Users/nicholas/Downloads/MT_CKD/LBLRTM/aer_rt_utils'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:AER-RC/aer_rt_utils.git' into submodule path '/Users/nicholas/Downloads/MT_CKD/LBLRTM/aer_rt_utils' failed
Failed to clone 'aer_rt_utils'. Retry scheduled
Cloning into '/Users/nicholas/Downloads/MT_CKD/LBLRTM/cross-sections'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:AER-RC/cross-sections.git' into submodule path '/Users/nicholas/Downloads/MT_CKD/LBLRTM/cross-sections' failed
Failed to clone 'cross-sections'. Retry scheduled
Cloning into '/Users/nicholas/Downloads/MT_CKD/LBLRTM/aer_rt_utils'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:AER-RC/aer_rt_utils.git' into submodule path '/Users/nicholas/Downloads/MT_CKD/LBLRTM/aer_rt_utils' failed
Failed to clone 'aer_rt_utils' a second time, aborting
Submodule path 'aer_rt_utils': checked out '13f3a75ddd98fddcf0c4414031acd0d321be8569'
fatal: Failed to recurse into submodule path 'LBLRTM'
```

**Comment by pernak18** [comment]
@Nicholaswogan i'll have to look into why `git` is pulling submodules with SSH instead of HTTPS when you have an HTTPS URL in your clone command. for now, a workaround maybe to generate a public SSH key and post it in [your GitHub SSH keys Settings](https://github.com/settings/keys)

**Comment by pernak18** [comment]
pretty sure this was addressed with 72c0c0d. i've successfully tested a recursive clone with both the SSH and HTTPS protocols.
